frontend/src/views/pricing-view.js

```diff
@@ -0,0 +1,158 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import '../components/ex-card.js';
+
+/**
+ * `pricing-view`
+ *
+ * PricingView element.
+ *
+ * @customElement
+ * @polymer
+ */
+class PricingView extends PolymerElement {
+
+    static get template() {
+        return html`
+            <style include="common-styles">
+                :host {
+                    display: block;
+                }
+                
+                .wrapper {
+                    margin: 0 auto; /* Horizontally center the layout grid if there is space around it */
+                    max-width: 960px;
+                    padding: 0 var(--lumo-space-m);
+                }
+
+                #header {
+                    padding: var(--lumo-space-wide-l);
+                    margin-bottom: var(--lumo-space-xl);
+                    border-bottom: 1px solid #dee2e6;
+                    box-shadow: var(--lumo-box-shadow-xs);
```

Comment:
Here the same thing as with the card: combining the border into the box shadow makes it visually better looking. Also, the border color should use a Lumo contrast color.

Reply:
Thanks. I don't know how that hex color got in there.

Reply:
Done.